Makefile

```diff
@@ -47,4 +53,4 @@ megalint:  ## Run the mega-linter.
 	docker run --platform linux/amd64 --rm \
 		-v /var/run/docker.sock:/var/run/docker.sock:rw \
 		-v $(shell pwd):/tmp/lint:rw \
-		oxsecurity/megalinter:v7
+		oxsecurity/megalinter:v8
```

Comment:
We should consider using an image optimised for Python based projects: https://megalinter.io/latest/flavors/python/

Reply:
Have updated to use the python specific version